### DISCUSSION TITLE
yhat, get forecast values

### DISCUSSION THREAD

**Discussion by mandar-avhad** [ourownstory/neural_prophet#517] [Q&A - get help using NeuralProphet] [answered]
Hi, after doing model.predict(df), I'm getting the following forecast. It has yhat1.yhat2....till the yhat'n' ('n' being the 'n'-th step forecasted into the future, n_forecasts steps)

Now, I want to get the final column of values that are being forecasted. Just like the 1 yhat column.
How can I get the one final forecast column? Please help.

![image](https://user-images.githubusercontent.com/61730638/149275477-ec0a2e7c-46ad-4dc5-b0cc-b4ebc0b23664.png)

@ourownstory

**Comment by mandar-avhad** [comment]
@ourownstory Hi, there?

**Comment by ourownstory** [comment, marked as answer]
@mandar-avhad 
Might this be of help?
https://github.com/ourownstory/neural_prophet/blob/master/tutorials/feature-use/collect_predictions.ipynb
